### DISCUSSION TITLE
Default OpenBox behaviour regarding window expansion on screen edge

### DISCUSSION THREAD

**Discussion by ghost** [lxqt/lxqt#2314] [General]
Hi there, I've settled on LXQT and I love how simple it is. I look forward to using it from now on for many years to come as I personally like it the most out of the other DMs I've tried. 

I find myself dragging windows onto the top screen edge intending to maximize them but they don't resize, so I wanted to start a discussion about this and find out the thoughts you all had on this and what work arounds if any you think are best to enable this. Would this be something you might consider adding as well as it retains simplicity but adds a tiny bit of usability that's been ingrained into I would imagine alot of people or is it also something I should maybe unlearn as well. Not sure but would love to hear your thoughts etc on it.

**Comment by tsujan** [comment]
> Would this be something you might consider adding

LXQt is window-manager agnostic. Openbox isn't an LXQt component; it can just be used with LXQt. LXQt doesn't ship any Openbox config.

**Reply by ghost** [reply]
> > Would this be something you might consider adding
> 
> LXQt is window-manager agnostic. Openbox isn't an LXQt component; it can just be used with LXQt. LXQt doesn't ship any Openbox config.

I see, I suspect this has more to do with my lack of understanding about how everything is put together for now. Thanks for the reply:)

**Reply by Chiitoo** [reply]
Indeed, if you use something like KWin instead, it does something like this, or at least I think I remember finding myself disabling such behaviour in the past.  ^^

**Reply by tsujan** [reply]
> or at least I think I remember finding myself disabling such behaviour in the past. ^^

Yes, KWin does it by default, for maximizing and tiling, depending on where the window is dragged to.
